src/kactl/numerical/SolveLinearBinary.h

```diff
@@ -1,4 +1,5 @@
 /**
+ * Name: Linear equations in Z\lower 1.5pt\hbox{\tenbf 2}
```

Comment:
Why not `$a_b$`?

Reply:
Because… Nevermind :innocent:. Will work after merging #60.